.circleci/config.yml

```
@@ -136,6 +136,9 @@ jobs:
       name: amphtml-medium-executor
     steps:
       - setup_vm
+      - run:
+          name: 'Install Bazel'
+          command: sudo apt-get install bazel
```

This didn't work as written: https://app.circleci.com/pipelines/github/ampproject/amphtml/5098/workflows/d84e4ced-5830-4bac-918a-628196cc8fc0/jobs/68452/parallel-runs/0/steps/0-108

Can we try this alternative (adapted from [here](https://circleci.com/blog/bazel-for-android/))? (We may not need to install `curl` or `gnupg`, so it may also be worth trying without it.)

```suggestion
      - run:
          name: 'Add Bazel Repo'
          command: |
            sudo apt install curl gnupg
            curl -fsSL https://bazel.build/bazel-release.pub.gpg | gpg --dearmor > bazel.gpg
            sudo mv bazel.gpg /etc/apt/trusted.gpg.d/
            echo "deb [arch=amd64] https://storage.googleapis.com/bazel-apt stable jdk1.8" | sudo tee /etc/apt/sources.list.d/bazel.list
      - run:
          name: 'Install Bazel'
          command: |
            sudo apt update
            sudo apt install bazel
```

---

Done in a new script.